settings.gradle.kts

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
We don't want the default "init" comment section here. It tells nothing.

Use the Apache 2.0 license headers instead. Again, please see how it's done in other examples, such as `kanban`.